spidermon/utils/web.py

```diff
@@ -0,0 +1,17 @@
+from urllib.parse import urlparse
+from urllib.request import urlopen
+
+
+def is_url(path: str) -> bool:
```

Comment:
Function annotations doesn't work in Python 2.7.

Reply:
We already have a function to verify if a string is a valid URL:
`spidermon.contrib.validation.utils.is_valid_url`

Reply:
`is_valid_url` doesn't allow s3 paths like s3://bucket/file and it has a different purpose - validate with schema.